common/ucf-faq-list-common.php

```diff
@@ -110,6 +110,58 @@ public static function display_faq( $post, $atts ) {
 		}
 
 
+		/**
+		 * Method to output the related faq question and answers with title.
+		 * @author RJ Bruneel
+		 * @since 1.0.0
+		 **/
+		public static function display_related_faqs( $posts, $title, $atts ) {
+			$retval = "";
+
+			ob_start();
+
+			if ( $posts ) :
+
+			?>
+				<h2 class="<?php UCF_FAQ_Config::add_athena_attr( 'h4 pt-4' ); ?>">
+					<?php echo $title; ?>
+				</h2>
+			<?php
+				$unique_id = wp_rand();
+
+				foreach ( $posts as $post ) {
+					echo UCF_FAQ_Common::display_faq( $post, $atts, $unique_id );
+				}
+
+				$retval = ob_get_clean();
+
+			endif;
+
+			return $retval;
+		}
+
+
+
+		/**
+		 * Method to output the footer CTA.
+		 * @author RJ Bruneel
+		 * @since 1.0.0
+		 **/
+		public static function display_footer_cta( $cta_text, $cta_url ) {
```

Comment:
Let's add the snippet below into this new function:
```php
if ( substr( $cta_url, 0, 1 ) === '/' ) {
 	$cta_url = site_url() . $cta_url;
}
```
and remove it from templates/taxonomy-topic.php, lines 33-35, so that relative and absolute URLs are supported wherever this function gets called